### ISSUE TITLE
guided-diffusion weights

### ISSUE THREAD

**Issue by afiaka87** [lucidrains/DALLE-pytorch#251]
https://github.com/openai/guided-diffusion/issues/1

Worth a shot, right?

**Comment by kobiso** [comment]
@afiaka87 
Do you have any idea how to use DDPM and ADM for DALLE?
I skimmed through the papers and I guess they don't have a process to generate discrete visual token, which are necessary for transformer training.
Also, ADM suggested to use classifier guidance with label information, when many of datasets for DALLE includes captions rather than label information.
I would like to hear your opinion :)

**Comment by crowsonkb** [comment]
I don't know how to use diffusion models with anything DALL-E related either and I'm not sure you can at all. I have managed to guide them with gradients from CLIP though.

**Comment by afiaka87** [comment]
We've had a user reach out in the discord with a plan to attempt this - re-opening.
